kubernetes/cosmos/values.yaml

```diff
@@ -37,32 +38,28 @@ rclone:
     tag: 1.45
     pullPolicy: IfNotPresent
 
+  suspend: true
   schedule: "0 */12 * * *"
   successfulJobsHistory: 1
 
   remote:
-    accessKey: my-access-key
-    secretKey: my-secret-key
-    endpoint: http://cloudserver
-    region: pfsd-1
-    bucket: pfsd-1
+    credentials:
+      existingSercret: ""
```

Comment:
Can prob just leave this commented out with a small note of what it can be used for